api/eds.proto

```diff
@@ -172,21 +172,32 @@ message UpstreamLocalityStats {
   // originate at the client and the endpoint responded successfuly. For gRPC,
```

Comment:
Can you fixup the `LoadAssignmentResponse.intervalStats` above to correct field name and add a link to where the field is defined?